runway/hooks/staticsite/auth_at_edge/templates/shared.py

```diff
@@ -0,0 +1,363 @@
+# pylint: skip-file
+"""Shared functionality for the Auth@Edge Lambda suite.
+
+NOTE: pylint: skip-file has been added to this file to avoid an issue with a false
+positive about duplicated code. Please remove during iteration.
```

Comment:
skip-file & this note should be ready to be dropped now that the duplicate filter has been expanded